### ISSUE TITLE
Accessibility - WCAG 1.3.1 Heading, there are no invisible headings anymore

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#134]
Headings make the structure of the document clear,
There are no invisible Headings anymore, like:
< h2> Orientierung innerhalb der Bundesverwaltung</h2> 
< h2> Sprachwahl</h2> 
< h2> Service-Navigation</h2> 
< h2> Suche</h2> 
etc.etc.

(Not visible Headings do help screen reader users for their orientation)
Please correct as this was there in the original version
